Build/Customizations/Projects.props

```diff
@@ -195,6 +195,7 @@
     <ReleaseProjectFiles Include="$(SolutionDirectory)\Remotion\ObjectBinding\Security\ObjectBinding.Security.csproj"/>
     <ReleaseProjectFiles Include="$(SolutionDirectory)\Remotion\ObjectBinding\Validation\ObjectBinding.Validation.csproj"/>
     <ReleaseProjectFiles Include="$(SolutionDirectory)\Remotion\ObjectBinding\Web\ObjectBinding.Web.csproj"/>
+    <ReleaseProjectFiles Include="$(SolutionDirectory)\Remotion\ObjectBinding\Web.ClientScript\ObjectBinding.Web.ClientScript.csproj"/>
```

Comment:
should put the Clientscript references first. right now, we don't have a dependency (i.e. building the nuget pacakge immediately after compiling a project, but that's not guaranteed.